codeship-steps.yml

```
@@ -11,3 +11,21 @@
   service: tracker
   tag: ^(main|develop)
   command: "true"
+
+- name: push_develop
+  service: app
+  type: push
+  image_name: silintl/serverless-mfa-api-go
+  image_tag: "latest"
+  tag: develop
```

For pushing develop and tagged versions, I think we normally tag the image by the branch name (which includes version tags):
```suggestion
  image_tag: "{{.Branch}}"
  exclude: main
```
... then reserve the tag "latest" for the `main` (or `master`) branch.

(An example of this is [idp-id-broker's codeship-steps.yml](https://github.com/silinternational/idp-id-broker/blob/develop/codeship-steps.yml) file.)

---

OK. I just didn't want it to push an image that hadn't been PR reviewed.

---

Ahh, that makes sense. If you want to limit it down like that, I think you can do something like `tag: (develop|main)`. I'm fine with either what's already here or with that approach. I don't typically see any value in pushing images for feature branches (typically), so feel free to make that change if you're so inclined. I'll approve either way.